R/subscript.R

```diff
@@ -360,10 +362,10 @@ cnd_subscript_action <- function(cnd, assign_to = TRUE) {
   }
 
   if (!is_string(action, subscript_actions)) {
-    abort(paste0(
-      "Internal error: `cnd$subscript_action` must be one of ",
-      "`subset`, `extract`, `assign`, `rename`, `remove`, or `negate`."
-    ))
+    cli::cli_abort(
+      "`cnd$subscript_action` must be one of {.or {.arg {subscript_actions}}}.",
+      .internal = TRUE
```

Comment:
I know about the `.internal` bug with `cli_abort()` that requires the dev version of cli to get the right `.frame`, but I'm not going to require dev cli here because I don't want vctrs to be blocked by it. We don't snapshot this or anything, so it shouldn't really matter